routes/viewRoute.js

```diff
@@ -39,6 +39,7 @@ router.get(
   viewController.getCreateRoomPage
 );
 
+router.get('/home', viewController.getHomePage);
```

Comment:
Read the issue properly, I said, add a home or landing page on  "/" URI, not on "/home" URI